sdks/python/apache_beam/runners/portability/portable_runner.py

```diff
@@ -84,16 +84,10 @@ def __init__(self):
   @staticmethod
   def default_docker_image():
     if 'USER' in os.environ:
-      if sys.version_info[0] == 2:
-        version_suffix = ''
-      elif sys.version_info[0:2] == (3, 5):
-        version_suffix = '3'
-      else:
-        version_suffix = '3'
-        # TODO(BEAM-7474): Use an image which has correct Python minor version.
-        logging.warning('Make sure that locally built Python SDK docker image '
-                        'has Python %d.%d interpreter. See also: BEAM-7474.' % (
-                            sys.version_info[0], sys.version_info[1]))
+      version_suffix = ''.join([str(i) for i in sys.version_info[0:2]])
```

Comment:
This changes the behavior for python 2. I agree with the change, let's make sure that we test it.

Reply:
`PortableRunnerTestWithLocalDocker` class at `portable_runner_test.py` will test this function. However, in order to run these tests, users should build docker images first, which I don't think all users need to do. Do we want to skip these tests at local? Is there any good practice I can refer?

Reply:
Could you chat with @alanmyrvold or @yifanzou on what would be the best testing practices here?

I think we should not skip tests if possible.

Reply:
+1 with Ahmet for not skipping these tests. But, failing a test which because of the user didn't build images beforehand does not make sense to me. Could we add warnings in the tests if the default images are missing?